bspump/http/web/server.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, app, id=None, config=None):
         self.aiohttp_app = aiohttp.web.Application(
             client_max_size=int(self.Config["max_body_size_bytes"])
         )
+        self.aiohttp_app.router.add_static("/static/", "./static", show_index=True)
```

Comment:
You need to be able to load the static files even when the pip package is installed.